src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PersonName.java

```diff
@@ -0,0 +1,119 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Example PNA+PAT+9435492908:OPI+++SU:KENNEDY+FO:SARAH+TI:MISS+MI:ANGELA'
+ */
+@Getter
+@Builder
+@EqualsAndHashCode(callSuper = false)
+@ToString
+public class PersonName extends Segment {
+
+    private static final String KEY = "PNA";
+    private static final String QUALIFIER = "PAT";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    private static final String FAMILY_NAME_QUALIFIER = "SU";
+    private static final String FIRST_NAME_QUALIFIER = "FO";
+    private static final String MIDDLE_NAME_QUALIFIER = "MI";
+    private static final String TITLE_QUALIFIER = "TI";
+
+    private final String nhsNumber;
+    private final PatientIdentificationType patientIdentificationType;
+    private final String surname;
+    private final String firstForename;
+    private final String title;
+    private final String secondForename;
```

Comment:
Should any of these be mandatory?

Reply:
All fields stay optional, added validation around identification and surname - at least one being required